deny.toml

```diff
@@ -37,8 +47,7 @@ license-files = [
 ]
 
 [bans]
```

Comment:
```suggestion
[bans]
# Deny multiple versions or wildcard dependencies.
multiple-versions = "deny"

skip-tree = [
    # rusoto_signature uses an older version of itoa
    { name = "rusoto_signature", version = "0.47.0" },
]
```

While I understand the sentiment, I think we need to be consistent with our other projects.